index.bs

```diff
@@ -833,7 +829,7 @@ url: https://w3c.github.io/permissions/#permissions-task-source; type: dfn;
 
 				1. For each |format| in |formats|["{{ClipboardUnsanitizedFormats/unsanitized}}"]:
 
-					1. If |format| is not in [=unsanitized data types=], then [=reject=] |p| with |format| {{"NotAllowedError"}} {{DOMException}} in |realm|.
+					1. If |format| is not in [=optional unsanitized data types=], then [=reject=] |p| with |format| {{"NotAllowedError"}} {{DOMException}} in |realm|.
```

Comment:
What's the reason this needs to be done in a separate for each loop here vs. inside the loop on line 866?

Reply:
In the Chromium's implementation we don't post the async task to read the data from the system clipboard for formats that aren't in `unsanitized` list if this check fails when `read()` is called. That makes sense as reading those formats are waste of resources if we end up throwing an exception anyway. This is why I had to move it outside the step that posts parallel tasks to do work.

Reply:
Ok, this sounds fine. It looks like we can bypass things like checking for permissions.